src/reducers/ModulesReducer.js

```diff
@@ -44,6 +44,7 @@ const initialState = () => {
 
   const usingInsurance = UIDatabase.objects('InsuranceProvider').length > 0;
   const usingPrescriptionCategories = UIDatabase.objects('PrescriptionCategory').length > 0;
+  const usingSupplierCreditCategories = UIDatabase.objects('SupplierCreditCategory').length > 0;
   const usingPaymentTypes = UIDatabase.objects('PaymentType').length > 0;
```

Comment:
Can we use the cash register if we're not using payment types? I have assumed that payment types need to be specified for new cash transactions to be valid.

Reply:
I mean if they haven't synced, or I guess if they don't want to use them